Tutorial-ETK_thorn-Interpolation_to_Arbitrary_Grids_multi_order.ipynb

```diff
@@ -1928,12 +1989,12 @@
    "outputs": [],
    "source": [
     "with open(os.path.join(Ccodesdir,\"src\",\"define_NumInterpFunctions.h\"), \"w\") as file:\n",
-    "    file.write(\"#define NumInterpFunctions \"+str(which_InterpCounter)+\"\\n\")"
+    "    file.write(\"#define NumInterpFunctions (\"+str(which_InterpCounter)+\"-4*enable_nuc_eos)\\n\")"
```

Comment:
This is actually doing the opposite what I intended it to do. I want it to subtract 4 whenever enable_nuc_eos is 0, not 1. I'll fix it.

Reply:
Yes, please fix. It seems more intuitive to add 4 when nuc_eos is enabled.

Reply:
I already fixed and pushed this on commit 0c9938a. Basically my solution was to use
`file.write(\"#define NumInterpFunctions (\"+str(which_InterpCounter)+\"-4*(1-enable_nuc_eos))`
since the variable `which_InterpCounter` is a Python variable and will already include the 4 additional functions.